.env.example

```diff
@@ -2,6 +2,6 @@ VITE_AUTH0_DOMAIN="auth0_domain"
 VITE_AUTH0_CLIENT_ID="auth0_client_id"
 VITE_AUTH0_AUDIENCE="auth0_audience"
 VITE_AUTH0_REDIRECT_URI="auth0_redirect_uri"
-VITE_GRPC_BASE_URL="localhost:3000"
+VITE_GRPC_BASE_URL="http://localhost:3000"
```

Comment:
This is somewhat unrelated to this PR but I get an error if I don't use this env setting.